addons/web_editor/static/src/js/wysiwyg/widgets/link.js

```diff
@@ -253,6 +253,10 @@ const Link = Widget.extend({
         if (!this.$link.attr('target')) {
             this.$link[0].removeAttribute('target');
         }
+        if(this.$link.hasClass("btn"))
+        {
+            this.$link.attr("data-oe-type","button");
+        }
```

Comment:
```suggestion
        if(this.$link[0].classList.contains("btn")) {
            this.$link[0].setAttribute("data-oe-type", "button");
        }
```

Not needed if we are going to manage this fix with ```oe_unbreakable``` class only